js/pro/cex.js

```diff
@@ -972,10 +972,11 @@ module.exports = class cex extends cexRest {
         const pair = this.safeString (data, 'pair', '');
         const symbol = this.pairToSymbol (pair);
         const storedOrderBook = this.safeValue (this.orderbooks, symbol);
+        const messageHash = 'orderbook:' + symbol;
```

Comment:
@pcriadoperez not directly related to this issue, but after upgrading to python 3.11 I can't use cex due to an issue with the `authenticate` method

```
CPConnector object at 0x105037810>
➜  ccxt git:(cex-catch-error) ✗ p cex watchOrderBook "BTC/USDT"
Python v3.11.2
CCXT v2.9.12
cex.watchOrderBook(BTC/USDT)
/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py:71: RuntimeWarning: coroutine 'ClientSession._ws_connect' was never awaited
  _cancel_all_tasks(loop)
RuntimeWarning: Enable tracemalloc to get the object allocation traceback
Traceback (most recent call last):
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 232, in <module>
    asyncio.run(main())
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 190, in run
    return runner.run(main)
           ^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/runners.py", line 118, in run
    return self._loop.run_until_complete(task)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 653, in run_until_complete
    return future.result()
           ^^^^^^^^^^^^^^^
  File "/Users/cjg/Git/ccxt10/ccxt/examples/py/cli.py", line 211, in main
    result = await method(*args)
             ^^^^^^^^^^^^^^^^^^^
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/pro/cex.py", line 811, in watch_order_book
    await self.authenticate()
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/pro/cex.py", line 1139, in authenticate
    self.spawn(self.watch, url, messageHash, self.extend(request, params), messageHash)
  File "/Users/cjg/Git/ccxt10/ccxt/python/ccxt/pro/base/exchange.py", line 89, in spawn
    task = self.asyncio_loop.create_task(method(*args))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/homebrew/Cellar/python@3.11/3.11.2_1/Frameworks/Python.framework/Versions/3.11/lib/python3.11/asyncio/base_events.py", line 436, in create_task
    task = tasks.Task(coro, loop=self, name=name, context=context)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Reply:
Actually it was not related to the python version and should be fixed now